modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -790,10 +789,9 @@
                       ]
                     },
                     "claimDate": {
-                      "type": [
-                        "string",
-                        "null"
-                      ],
+                      "format": "date",
+                      "type": "date",
+                      "nullable": true,
                       "description": "The date a claim was filed",
                       "example": "5/17/20"
```

Comment:
There are a few claimDate examples like above that use MM/DD/YY format instead of YYYY-MM-DD, just wanted to confirm that was intentional.